etc/systemd/system/linstor-satellite.service.d/override.conf

```diff
@@ -1,5 +1,5 @@
 [Service]
-Environment=LS_KEEP_RES=^xcp-persistent*
+Environment=LS_KEEP_RES=^xcp-persistent.*
```

Comment:
The new regexp will still match `xcp-persistenttttttt`. Can't it be more precise?

Reply:
Is there actually an issue in being too broad? (Genuinely asking) If so, we could use a more precise regex like `xcp-persistent-([a-z0-9_]+-?)+` or `xcp-persistent(-[a-z0-9_]+)+` (both provides the same results). This would restrict matches to actually valid strings instead of `xcp-persistenttttttt` or `xcp-persistent-`, `xcp-persistent-database------`, etc.

Reply:
That's why I approved in the first place. I'm ok with the broad regex, my suggestion was more to add a more precise proposition following Samuel's question. But the original proposition you made is ok for me.